debian/changelog.in

```diff
@@ -1,10 +1,4 @@
-journalpump (2.0.0) unstable; urgency=low
```

Comment:
debian package version is autogenerated, and the current tags produce a smaller version than 2.0.0 which in turn causes debian package build to fail.